Modix/ClientApp/src/models/moderation/RoleDesignation.ts

```diff
@@ -1,5 +1,6 @@
 export enum RoleDesignation
 {
     Rank = "Rank",
-    ModerationMute = "ModerationMute"
-}
\ No newline at end of file
+    ModerationMute = "ModerationMute",
+    ResctrictedMentionability = "ResctrictedMentionability"
```

Comment:
Small spelling error here, should be "RestrictedMentionability" (I like the new word, btw).